TalkAloud/TalkAloud/ViewControllers/AudioRecordingsTableViewController.swift

```diff
@@ -46,11 +46,45 @@ class AudioRecordingsTableViewController: UITableViewController {
         return true
     }
     
-    override func tableView(_ tableView: UITableView, commit editingStyle: UITableViewCell.EditingStyle, forRowAt indexPath: IndexPath) {
-        if editingStyle == .delete {
+    override func tableView(_ tableView: UITableView, trailingSwipeActionsConfigurationForRowAt indexPath: IndexPath) -> UISwipeActionsConfiguration? {
+        let audioCell = tableView.dequeueReusableCell(withIdentifier: "audio", for: indexPath)
+        let deleteAction = UIContextualAction(style: .destructive, title: "Delete") { (action, view, completionHandler) in
             AudioManager.sharedInstance.removeFile(at: indexPath.row)
             tableView.deleteRows(at: [indexPath], with: .automatic)
         }
+        
+        let editAction = UIContextualAction(style: .normal, title: "Edit") { (action, view, completionHandler) in
+            let ac = UIAlertController(title: "Change name", message: nil, preferredStyle: .alert)
+            ac.addTextField()
+            
+            let renameCellAction = UIAlertAction(title: "Done", style: .default) { [unowned ac] action in
+                let newCellName = ac.textFields?[0].text
+                
+                if let newCellName = newCellName {
```

Comment:
Rename and clean the logic here.
You are not changing the cell, because the cell is driven by the data source. Not the other way around.